client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -376,7 +357,6 @@ export const useCommandsArrayWpcom = ( {
 		{
 			name: 'managePHPVersion',
 			label: __( 'Manage PHP version' ),
-			searchLabel: __( 'manage PHP issue' ),
```

Comment:
I assume that `searchLabel` is the same as `label` here, too, and it was just a typo.

Reply:
Yeah it should be the same, likely a typo!